packages/badge/tests/colorCodedBadge.test.tsx

```diff
@@ -10,12 +10,16 @@ expect.addSnapshotSerializer(serializer);
 
 describe("ColorCodedBadge", () => {
   it("renders with a color", () => {
-    const component = shallow(<ColorCodedBadge color="#f00f00" />);
+    const component = shallow(
+      <ColorCodedBadge color="#f00f00">Color Coded Badge</ColorCodedBadge>
+    );
     expect(toJson(component)).toMatchSnapshot();
   });
   it("renders with an icon and color", () => {
     const component = shallow(
-      <ColorCodedBadge color="#f00f00" iconShape={SystemIcons.CircleCheck} />
+      <ColorCodedBadge color="#f00f00" iconShape={SystemIcons.CircleCheck}>
+        Color Coded Badge with icon
+      </ColorCodedBadge>
```

Comment:
does this change reflect a change in how the component works? If I read correctly, it always worked this way, you're just improving the test. Wanted to double check. Maybe change one test to take children and leave the other one without?

Reply:
on second thought, if you want to test that it renders the child correctly, I'd add a new test entirely and leave these alone. Would give us greater confidence in the refactor